cafebabel/templates/base.html

```diff
@@ -56,15 +56,12 @@
     </ul>
 </nav>
 
-<button class="menu follow">Follow</button>
-
-<nav>
+<nav id=social-networks>
     <ul>
-        <li><a href=#>Facebook</a></li>
-        <li><a href=#>Twitter</a></li>
-        <li><a href=#>Instagram</a></li>
-        <li><a href=#>LinkedIn</a></li>
-        <li><a href=#>Youtube</a></li>
+        <li><a href=# class=icon-facebook></a></li>
```

Comment:
Still wondering what is the indent size on this project for HTML. Do you go for 2 or 4 spaces?

Reply:
2, but I am not going to re-indent manually all the files we have been working on before we set the indent to 2, am I? @vinyll

Reply:
Possible to add review so that I can merge? @vinyll @davidbgk